lib/commands/addValue.js

```diff
@@ -67,6 +68,7 @@ module.exports = function addValue (selector, value) {
 
         return this.unify(elementIdValueCommands);
 
-    });
+    })
+    .catch(staleElementRetry.bind(this, 'isVisible', arguments));
```

Comment:
e.g. here, it should be `.catch(staleElementRetry.bind(this, 'addValue', arguments));`